Documentation/toc.yml

```diff
@@ -230,12 +230,14 @@
       href: README_Button.md
     - name: Bounds Control
       href: README_BoundsControl.md
-    - name: Bounding Box [Obsolete]
-      href: README_BoundingBox.md
     - name: Object Manipulator
       href: README_ObjectManipulator.md
     - name: Constraint Manager
       href: README_ConstraintManager.md
+    - name: Elastic System [Experimental]
```

Comment:
I'm wondering if we should put the experimental elastic system down the list and group it with other experimental features. Also not sure if we should group the obsolete ones as well.

Reply:
moved to other experimental ux and also moved obsolete components below experimental ones